Sming/include/user_config.h

```diff
@@ -6,7 +6,7 @@ extern "C" {
 #endif
 
 	// UART config
-	#define SERIAL_BAUD_RATE 115200
+	#define SERIAL_BAUD_RATE COM_SPEED_SERIAL
```

Comment:
Default is now left at 115200. The fix just removes hardcoded value and takes into consideration COM_SPEED_SERIAL which can be given as make parameter to override

Makefile-project.mk
```
## COM port parameters
# Default COM port speed (generic)
COM_SPEED ?= 115200

# Default COM port speed (used for flashing)
COM_SPEED_ESPTOOL ?= $(COM_SPEED)

# Default COM port speed (used in code)
COM_SPEED_SERIAL  ?= $(COM_SPEED)
```